modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/connection/ElasticsearchConnectionManager.java

```diff
@@ -275,6 +283,12 @@ public void onElasticsearchConfigurationUpdate() {
 	}
 
 	public void removeElasticsearchConnection(String connectionId) {
+		if (_log.isInfoEnabled()) {
+			_log.info(
+				"Start Removing Elasticsearch Connection with connection ID: " +
+					connectionId);
+		}
+
 		if (connectionId == null) {
 			return;
 		}
```

Comment:
Could we log what the _elasticsearchConnectionSupplier returns and if it's null.